source/core/NumericString.cpp

```diff
@@ -1986,15 +1972,55 @@ VIREO_FUNCTION_SIGNATUREV(StringScan, StringScanStruct)
         MakeFormatString(tempFormat.Value, &errorCluster, argCount, arguments);
         format.AliasAssignCStr(reinterpret_cast<ConstCStr>(tempFormat.Value->Begin()));
     }
-    newOffset += FormatScan(&input, &format, argCount, arguments);
-    
+    newOffset += FormatScan(&input, &format, argCount, arguments, null);
+
     _Param(OffsetPast) = newOffset;
     _Param(StringRemaining)->Resize1D(input.Length());
     TypeRef elementType = _Param(StringRemaining)->ElementType();
     elementType->CopyData(input.Begin(), _Param(StringRemaining)->Begin(), input.Length());
     return _NextInstruction();
 }
+//------------------------------------------------------------
+struct StringScanStructWithErr : public VarArgInstruction
+{
+    _ParamDef(StringRef, StringInput);
+    _ParamDef(StringRef, StringRemaining);
+    _ParamDef(StringRef, StringFormat);
+    _ParamDef(UInt32, InitialPos);
+    _ParamDef(UInt32, OffsetPast);
+    _ParamDef(ErrorCluster, ErrClust);
+    _ParamImmediateDef(StaticTypeAndData, argument1[1]);
+    NEXT_INSTRUCTION_METHODV()
+};
+//------------------------------------------------------------
+VIREO_FUNCTION_SIGNATUREV(StringScanWithErr, StringScanStructWithErr)
```

Comment:
I'm going to do this in a separate PR because I like to do things incrementally.
Deleting two prims is not "doing work twice", and I'm doing it, not you.